app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -36,6 +37,7 @@ import java.security.InvalidParameterException
 /**
  * A simple wrapper for SharedPreferences that makes reading preference a little bit easier.
  */
+@AnyThread
```

Comment:
nit: I don't think this clearly means, "Thread Safe" to most people (I've gotten questions about my use of `@UiThread` annotations and I think those are clearly). I think we should comment "thread safe" in the kdoc.

Reply:
Could we possibly refer to the Android Doc? Where I agree we can put in the kdoc, I think this at least makes it clear to anyone with the source code too since you can command+click into the code 😄